shared/common-adapters/avatar.js

```diff
@@ -221,104 +204,57 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
   }
 }
 
-export type {AvatarSize}
-const realConnector = compose(
```

Comment:
instead of having 2 connected avatars, lets just have one and use a flag to include if you want the following dots or not